packages/react/src/table/TableRow.tsx

```diff
@@ -18,15 +19,38 @@ export function TableRow({
 	invalid,
 	selected,
 }: TableRowProps) {
-	const { tableLayout } = useTableContext();
+	const { frozenColumnsOffsets, striped, tableLayout } = useTableContext();
+
+	const frozenColumnStyles = frozenColumnsOffsets?.reduce(
+		(acc, { columnNumber, offsetValue, isLastColumn }) => {
+			return {
+				[minContainerBreakpointForFrozenColumns]: {
+					...acc[minContainerBreakpointForFrozenColumns],
+					[`& :nth-child(${columnNumber}):where(td, th)`]: {
```

Comment:
Emotion complains about `nth-child` but it seems fine as no `style` tags are inserted and the component can't be customised.